app.py

```diff
@@ -832,7 +833,7 @@ def ptb_to_ptree(ptb_tree: str) -> tuple[ParentedTree, list]:
 	ptree, senttok = brackettree(ptb_tree)
 	for subt in ptree.subtrees():
 		for e in PUNCT_ESCAPING:
-			# we implement special escaping of punctuation tags `LRB`, `RRB`, and `HYPH` for the parser
+			# we implement special escaping of punctuation tags `LRB`, `RRB`, `HYPH`, and `COL` for the parser
```

Comment:
Hmm, I thought `:` was a valid tag in PTB. Is the issue that ":-p" contains a hyphen? If so maybe `HYPH` should apply to all puntuation tokens containing a hyphen?